test/plugin/test_out_google_cloud.rb

```diff
@@ -176,6 +178,7 @@ def test_managed_vm_metadata_loading
   end
 
   def test_gce_metadata_does_not_load_when_fetch_gce_metadata_is_false
```

Comment:
Oh OK, so we doing the opposite and are using those stubs to check assert_not_requested. 
Makes sense now :)